hasura/metadata/databases/default/tables/public_app_metadata.yaml

```diff
@@ -0,0 +1,203 @@
+        - is_developer_allow_listing
+        - integration_url
+        - app_website_url
+        - source_code_url
+        - verified_at
+        - review_message
+        - status
+        - updated_at
+      filter:
+        app:
+          team:
+            id:
+              _eq: X-Hasura-Team-Id
+  - role: reviewer
+    permission:
+      columns:
+        - id
+        - app_id
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - world_app_description
+        - category
+        - is_developer_allow_listing
+        - integration_url
+        - app_website_url
+        - source_code_url
+        - verified_at
+        - reviewed_by
+        - is_reviewer_app_store_approved
+        - is_reviewer_world_app_approved
+        - review_message
+        - status
+        - created_at
+        - updated_at
+      filter:
+        status:
+          _eq: awaiting_review
+  - role: service
+    permission:
+      columns:
+        - name
+        - logo_img_url
+      filter:
+        status:
+          _in: [unverified, verified]
+  - role: user
+    permission:
+      columns:
+        - app_id
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - world_app_description
+        - category
+        - is_developer_allow_listing
+        - integration_url
+        - app_website_url
+        - source_code_url
+        - verified_at
+        - review_message
+        - status
+        - updated_at
+      filter:
+        app:
+          team:
+            users:
+              id:
+                _eq: X-Hasura-User-Id
+update_permissions:
+  - role: api_key
+    permission:
+      columns:
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - category
+        - integration_url
+        - is_developer_allow_listing
+        - world_app_description
+        - app_website_url
+        - source_code_url
+      filter:
+        _and:
+          - status:
+              _eq: unverified
+          - app:
+              team:
+                id:
+                  _eq: X-Hasura-Team-Id
+      check: {}
+  - role: reviewer
+    permission:
+      columns:
+        - verified_at
+        - reviewed_by
+        - is_reviewer_app_store_approved
+        - is_reviewer_world_app_approved
+        - review_message
+        - status
+      filter:
+        status:
+          _eq: awaiting_review
+      check: {}
+  - role: user
+    permission:
+      columns:
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - category
+        - integration_url
+        - is_developer_allow_listing
+        - world_app_description
+        - app_website_url
+        - source_code_url
+      filter:
+        _and:
+          - status:
+              _eq: unverified
```

Comment:
for a subsequent improvement but this will also need permissions to change the status back to unverified if it's awaiting_review (to pull back from review)

Reply:
Ok will make this part of a new commit when I start adding the review status. Probably like 2 merges away